rosdep/python.yaml

```diff
@@ -173,6 +173,10 @@ cython3:
   gentoo: [dev-python/cython]
   nixos: [python3Packages.cython]
   ubuntu: [cython3]
+dm-env-pip:
```

Comment:
Please name the key `python3-dm-env-pip`: https://github.com/ros/rosdistro/blob/master/CONTRIBUTING.md#pip-1

Reply:
Resolved in [e14276c](https://github.com/ros/rosdistro/pull/40747/commits/e14276cb6e5120e62dc6f5935af7f2525d4cf78e).